ihmc-high-level-behaviors/src/main/java/us/ihmc/perception/RapidHeightMapThread.java

```diff
@@ -21,34 +30,43 @@
 public class RapidHeightMapThread extends RepeatingTaskThread
 {
    private final DepthImageBodyCollisionFilter bodyCollisionFilter;
+   private final DepthImageFlyingPointsFilter flyingPointsfilter;
```

Comment:
Don't apply this through GitHub, but here's my suggestion: (Capitalize the 2nd 'f')
```suggestion
   private final DepthImageFlyingPointsFilter flyingPointsFilter;
```